e107_plugins/forum/templates/forum_viewtopic_template.php

```diff
@@ -481,6 +481,14 @@
 
 									";
 
+$FORUM_VIEWTOPIC_TEMPLATE['quickreply'] = "
+						<div class='form-group'>
+							<textarea cols='80' placeholder='{LAN=FORUM_2007}' rows='4' id='forum-quickreply-text' class='tbox input-xxlarge form-control' name='post' onselect='storeCaret(this);' onclick='storeCaret(this);' onkeyup='storeCaret(this);'></textarea>
+						</div>
+						<div class='center text-center form-group'>
+							{QR_SBUTTON:value=".LAN_FORUM_2007."}
```

Comment:
Best to just make LAN_FORUM_2007 the default (within the shortcode) unless $parm['label'] is found.